nixos/modules/services/monitoring/prometheus/exporters/sql.nix

```diff
@@ -0,0 +1,28 @@
+{ config, lib, pkgs, options }:
+with lib;
+let
+  cfg = config.services.prometheus.exporters.sql;
+in
+{
+  extraOpts = {
+    configFile = lib.mkOption {
+      type = with types; nullOr path;
```

Comment:
Not sure if this should be made required, i.e. simply `path`. In the future the module might get extended with yaml generation via Nix configuration, then we would need `nullOr path` anyway.